config/config.go

```diff
@@ -26,6 +26,38 @@ func NewConfig() Configuration {
 }
 
 func setConfigDefaults(v *viper.Viper) {
+	v.SetDefault("port", 2424)
+	v.SetDefault("admin_port", 2525)
+	v.SetDefault("log.level", "info")
+	v.SetDefault("backend.type", "memory")
+	v.SetDefault("backend.aerospike.host", "")
+	v.SetDefault("backend.aerospike.port", 0)
+	v.SetDefault("backend.aerospike.namespace", "")
+	v.SetDefault("backend.aerospike.default_ttl_seconds", 0)
+	v.SetDefault("backend.azure.account", "")
+	v.SetDefault("backend.azure.key", "")
+	v.SetDefault("backend.cassandra.hosts", "")
+	v.SetDefault("backend.cassandra.keyspace", "")
+	v.SetDefault("backend.memcache.hosts", []string{""})
+	v.SetDefault("backend.redis.host", "")
+	v.SetDefault("backend.redis.port", 0)
+	v.SetDefault("backend.redis.password", "")
+	v.SetDefault("backend.redis.db", 0)
+	v.SetDefault("backend.redis.expiration", 0)
+	v.SetDefault("backend.redis.tls.enabled", false)
+	v.SetDefault("backend.redis.tls.insecure_skip_verify", false)
+	v.SetDefault("compression.type", "")
```

Comment:
This should default to "snappy" to match the config.yaml.

Reply:
I did notice the "snappy" value for the compression in our `config.yaml` file. But, should it be set as default? The alternative is `"none"` that won't give the compression overhead.

Reply:
Is config.yaml not considered our default values right now? I think having them disagree would be confusing. I vote to keep it as "snappy". Other thoughts?

Reply:
Done